libs/langgraph/src/pregel/index.ts

```diff
@@ -1107,6 +1107,70 @@ export class Pregel<
           saved ? saved.metadata : undefined
         );
       }
+
+      if (asNode === "__input__") {
```

Comment:
good idea to use the values in `constants.ts` for this:

```suggestion
      if (asNode === INPUT) {
```

might as well get the other magic strings elsewhere in this method as well.

Reply:
The main reason why `bulkUpdateState` exists is that `updateState` only supports accepting a single update for checkpoint, which won't work for map-reduce use cases. It's currently used only internally, so I'm fine with marking this with `@internal` and/or hiding this from public API.

Haven't attempted to break down the method in the previous PR (https://github.com/langchain-ai/langgraphjs/pull/969), instead tried to make as few changes as possible. Will try to do so in a different PR.